mteb/tasks/Classification/fin/FinToxicityClassification.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class FinToxicityClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="FinToxicityClassification",
+        description="""
+        This dataset is a DeepL -based machine translated version of the Jigsaw toxicity dataset for Finnish. The dataset is originally from a Kaggle competition https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/data.
+        The original dataset poses a multi-label text classification problem and includes the labels identity_attack, insult, obscene, severe_toxicity, threat and toxicity.
+        Here adapted for toxicity classification, which is the most represented class.
+        """,
+        dataset={
+            "path": "TurkuNLP/jigsaw_toxicity_pred_fi",
+            "revision": "6e7340e6be87124f319e25290778760c14df64d3",
+        },
+        reference="https://aclanthology.org/2023.nodalida-1.68",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["fin-Latn"],
+        main_score="f1",
+        date=("2023-03-13", "2023-09-25"),
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
```

Comment:
Since it is a toxicity classification, I would say it is more like Sentiment/Hate as in task metadata

Reply:
Indeed, fixed!